src/single_random_draw.rs

```diff
@@ -0,0 +1,199 @@
+//! This library provides efficient algorithms to compose a set of unspent transaction outputs
+//! (UTXOs).
+
+#[cfg(any(test, feature = "rand"))]
```

Comment:
Maybe put the feature gate on the module itself?

Reply:
good idea.  done.